### PR TITLE
Update netex_equipmentPlace_version.xsd

### PR DESCRIPTION
**Pull request by ue71603**
EquipmentPlace extends now SiteElement

### CODE REVIEW

**Comment by ue71603** [comment]
Replaces [[[[<xsd:include schemaLocation="../../netex_part_1\part1_ifopt\netex_ifopt_site_version.xsd"/> ](https://github.com/NeTEx-CEN/NeTEx/pull/275)](https://github.com/NeTEx-CEN/NeTEx/pull/275)](https://github.com/NeTEx-CEN/NeTEx/pull/275)](https://github.com/NeTEx-CEN/NeTEx/pull/275)

**Comment by ue71603** [comment]
@skinkie  @Aurige  :  What is blocking your review?

File: xsd/netex_framework/netex_reusableComponents/netex_equipmentPlace_version.xsd [hunk 1]
```diff
@@ -95,6 +96,12 @@ Rail transport, Roads and Road transport
 							<xsd:group ref="ZoneGroup"/>
 						</xsd:sequence>
 						<xsd:group ref="PlaceGroup"/>
+<!--CD PROPOSAL -->
```
Comment (skinkie) [inline xsd/netex_framework/netex_reusableComponents/netex_equipmentPlace_version.xsd:99]: These kind of things.

Comment (Aurige) [reply xsd/netex_framework/netex_reusableComponents/netex_equipmentPlace_version.xsd:99]: just cleaning the comments should be Ok

File: xsd/netex_framework/netex_reusableComponents/netex_equipmentPlace_version.xsd [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <!-- edited with XMLSpy v2008 (http://www.altova.com) by Nicholas Knowles Knowles (Kizoom Ltd) -->
 <xsd:schema xmlns:siri="http://www.siri.org.uk/siri" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns="http://www.netex.org.uk/netex" targetNamespace="http://www.netex.org.uk/netex" elementFormDefault="qualified" attributeFormDefault="unqualified" version="1.0" id="netex_equipmentPlace_version">
 	<xsd:include schemaLocation="netex_equipment_version.xsd"/>
-	<xsd:include schemaLocation="../../netex_framework/netex_genericFramework/netex_place_version.xsd"/>
+	<xsd:include schemaLocation="../../netex_part_1/part1_ifopt/netex_ifopt_site_version.xsd"/>
```
Comment (skinkie) [inline xsd/netex_framework/netex_reusableComponents/netex_equipmentPlace_version.xsd:5]:
```suggestion
	<xsd:include schemaLocation="../../netex_part_1/part1_ifopt/netex_ifopt_site_version.xsd"/>
```

File: xsd/netex_framework/netex_reusableComponents/netex_equipmentPlace_version.xsd [hunk 3]
```diff
@@ -113,13 +117,14 @@ Rail transport, Roads and Road transport
...
 		<xsd:complexContent>
-			<xsd:extension base="Place_VersionStructure">
+			<xsd:extension base="SiteElement_VersionStructure"> 
 				<xsd:sequence>
 					<xsd:group ref="EquipmentPlaceGroup"/>
 				</xsd:sequence>
 			</xsd:extension>
 		</xsd:complexContent>
 	</xsd:complexType>
+
```
Comment (skinkie) [inline xsd/netex_framework/netex_reusableComponents/netex_equipmentPlace_version.xsd:127]:
```suggestion
```

**Review by skinkie — APPROVED** [review]

**Comment by skinkie** [comment]
@ue71603 is this a master or a next thing?

**Comment by ue71603** [comment]
@skinkie  This was a bug and differently documented, so I think master.

**Comment by ue71603** [comment]
@Aurige  Can you do a review please?

**Review by Aurige — APPROVED** [review]

File: xsd/netex_framework/netex_reusableComponents/netex_equipmentPlace_version.xsd [hunk 4]
```diff
@@ -113,7 +117,7 @@ Rail transport, Roads and Road transport
 			<xsd:documentation>Type for an EQUIPMENT PLACE.</xsd:documentation>
 		</xsd:annotation>
 		<xsd:complexContent>
-			<xsd:extension base="Place_VersionStructure">
+			<xsd:extension base="SiteElement_VersionStructure">
```
Comment (skinkie) [inline xsd/netex_framework/netex_reusableComponents/netex_equipmentPlace_version.xsd:120]:
```suggestion
			<xsd:extension base="SiteElement_VersionStructure">
```

**Pull request merged**